fetch4fd/nls/fetch4fd.fr-UTF-8

```diff
@@ -0,0 +1,12 @@
+{Mémoire : }
+{Mo RAM}
+{Disques installés : }
+{libre}
+{Version de DOS rapportée : }

```

Comment:
Odd encoding for a UTF-8 file here?

Reply:
So I'm really confused  now, so maybe it's some github thing. The changes in the PR overview https://github.com/shidel/fd-nls/pull/187/files still look strange. To be sure I checked out @cardpuncher 's branch to my local machine and they look fine.